SciPy2024/module2/3-retrieval-augmented-text-generation.ipynb

```diff
@@ -9,79 +9,22 @@
    ]
```

Comment:
<p>Include a link to this article as a reference <a href="https://www.nlplanet.org/course-practical-nlp/01-intro-to-nlp/11-text-as-vectors-embeddings" target="_blank">1.12 Representing Texts as Vectors: Word Embeddings — Practical NLP with Python (nlplanet.org)</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-ssec/tutorials/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>